src/tyr/route_serializer_osrm.cc

```diff
@@ -379,6 +378,30 @@ struct IntersectionEdges {
   }
 };
 
+std::string turn_lane_direction(uint16_t turn_lane) {
+  switch (turn_lane) {
+    case kTurnLaneReverse:
+      return "uturn";
```

Comment:
should we expand to have `left uturn` and `right uturn` so user can provide proper icon?

Reply:
Won't that break parsing by existing clients that are expecting a `uturn`? The strings here need to be a sub-set of strings set in the [`intersection` function](https://github.com/valhalla/valhalla/blob/master/src/tyr/route_serializer_osrm.cc#L572-L608), which does not have a left/right uturn. 

I'm half a mind about changing the `intersections(...)` to use this new function so that we don't have to manage strings in 2 places. What do you think?